external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -97,159 +108,123 @@ class KafkaSinkStreamingSuite extends KafkaSinkSuiteBase with StreamTest {
       .map(_._2)
 
     try {
-      input.addData("1", "2", "3", "4", "5")
-      failAfter(streamingTimeout) {
-        writer.processAllAvailable()
-      }
-      checkDatasetUnorderly(reader, 1, 2, 3, 4, 5)
-      input.addData("6", "7", "8", "9", "10")
-      failAfter(streamingTimeout) {
-        writer.processAllAvailable()
-      }
-      checkDatasetUnorderly(reader, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
+      input.addData(1, 2, 3, 4, 5)
+      verifyResult(writer)(checkDatasetUnorderly(reader, 1, 2, 3, 4, 5))
+      input.addData(6, 7, 8, 9, 10)
+      verifyResult(writer)(checkDatasetUnorderly(reader, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10))
     } finally {
       writer.stop()
     }
   }
 
-  test("streaming - write aggregation w/o topic field, with topic option") {
```

Comment:
We simplify the query because aggregation is not needed for testing origin intention, as well as aggregation is not supported in continuous mode.